e2e/wdio/platform/tests/search.e2e-spec.ts

```diff
@@ -1,14 +1,149 @@
-import { CheckboxGroupPO } from '../pages/checkbox-group.po';
-import { refreshPage } from '../../driver/wdio';
+import {
+    browserIsIE,
+    click,
+    getAttributeByNameArr,
+    getElementArrayLength,
+    getElementSize,
+    getText,
+    getTextArr,
+    isEnabled,
+    refreshPage,
+    setValue,
+    waitElementToBePresentInDOM,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import { SearchPo } from '../pages/search.po';
+import searchPageContent from '../fixtures/appData/search-page-content';
 
-describe('Search field', () => {
-    const checkboxGroupPage = new CheckboxGroupPO();
+describe('Search field', function() {
+    const searchPo: SearchPo = new SearchPo();
```

Comment:
try to use object destructuring

Reply:
TBD like `const {searchFields, searchIcons, open, etc.} = new SearchPo();`

Reply:
done